tenable_io/helpers/base.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+from tenable_io.config import TenableIOConfig
+
+
+class BaseHelper(object):
```

Comment:
After our discussion, it seems that `wait_until` is also used in `*Ref` classes as well. To keep it DRY, instead of extending `BaseHelper`, let's move it into `tenable_io.util` and import from it.